creating-virtual-machines/dedicated-cpu.adoc

```diff
@@ -42,8 +42,8 @@ pinned to the pCPUS that has been dedicated for the virt-launcher
 container.
 
 Expressing the desired amount of VMI’s vCPUs can be done by either
-setting the `spec.domain.cpu.cores` or
-`spec.domain.resources.requests.cpu` or
+setting the `spec.domain.cpu.sockets` or `spec.domain.cpu.cores` or
+ `spec.domain.cpu.threads` or `spec.domain.resources.requests.cpu` or
 `spec.domain.resources.limits.cpu` to a whole number, integer (e.g. 1,
 2, etc) indicating the number of vCPUs requested for the VMI.
 
```

Comment:
The notes below should probably be updated as well. (to sockets*cores*threads == to resources.cpu)

Reply:
Also add a Note that will explain the performance advantage of using sockets.